demo/components/presentational/__snapshots__/Land.test.js.snap

```diff
@@ -0,0 +1,24 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Land renders the component 1`] = `
+<Component
+  style={
+    Object {
+      "width": 224,
+    }
+  }
+>
+  <Component
```

Comment:
Yeah, it is interesting that it just calls them `Component`. I forget, have you found a solution to this yet?

Reply:
Snapshots don’t recognize some react-native components (i.e., View, Image are saved as Component)
**Issue**: facebook/react-native#22417
**Status**: Solved on react-native v0.57.8 (2018/12/13) release note
**Solution**: Wait for new Expo 32 release with RN v0.57.8.